### ISSUE TITLE
[Question] How to view the CosmosDB -> collections -> documents?

### ISSUE THREAD

**Issue by dinbtechit** [microsoft/azure-tools-for-java#6987]
How to view the CosmosDB -> collections -> documents?

It shows the collection, "users" but cannot see the documents within the users. I would like to see the contents of the json file in intellij. 

![image](https://user-images.githubusercontent.com/17984781/193843416-d15bfcc5-7724-4e97-b359-9b2c66b2e6fc.png)

**Comment by wangmingliang-ms** [comment]
@dinbtechit Data visualization is possible with the help of other plugins or other tools, e.g.
* manage with IDEA's `Database and SQL` plugin (if your IDEA is **Ultimate** edition).
![image](https://raw.githubusercontent.com/microsoft/azure-tools-for-java/endgame-202208/PluginsAndFeatures/azure-toolkit-for-intellij/azure-intellij-plugin-lib/src/main/resources/whatsnew.assets/202208.datasource.gif )
* manage with the `Data Explorer` in Azure Portal:
![image](https://user-images.githubusercontent.com/69189193/194022494-0c5b864d-0367-4a79-b928-e3abdf0f9aea.png)

**Comment by dinbtechit** [comment]
Thank you!

**Comment by wangmingliang-ms** [comment]
Hi, @dinbtechit We take your feature request very seriously. Our latest v3.70.0 released last Friday added some basic data management capabilities to Cosmos DB for `MongoDB` and `SQL` (and Azure Storage Accounts) in Azure Explorer. You can try it after the upgrading, I hope you can give us more suggestions, thank you!
![image](https://user-images.githubusercontent.com/69189193/200160660-57f08730-5719-476c-8e84-16e0bf3e3286.png)

**Comment by dinbtechit** [comment]
Thank you. This looks excellent.  Few minor UX recommendations:

1. When creating a cosmos DB - Instead of just saying `+ Create   ⌘N` for everything.. it would be nice if it had more contextual names.
![image](https://user-images.githubusercontent.com/17984781/200179031-a832ef0e-1e81-4e61-952b-a368e162aaa3.png)

   <br> For Example:
    - `+ Create Database`
         - `+ Create Collection` (this would be `+ Create Container` for sql databases)
              - `+ Create Document`
    - etc, etc  
2. (Nice to have but very minor) - When deleting and creating a database. Show approximate processing times: (For example - Percentage of completion) instead of indefinitely spinning - Animated Loading icon.
     
    ![image](https://user-images.githubusercontent.com/17984781/200179167-2dac5978-6f1c-4abb-9a0b-cfcf00684dd4.png)

3. You can only import document - There is no option to create a document.
    
    ![image](https://user-images.githubusercontent.com/17984781/200179376-37724847-3c7a-499f-9d3d-c24db0733a58.png)

    when you attempt to import this json document you get the following error:

    ![image](https://user-images.githubusercontent.com/17984781/200179490-31278924-1572-4e92-983b-a12247544c74.png)

4. When viewing a document there needs to be an option to switch between a table or json format.

**Comment by Flanker32** [comment]
@dinbtechit Thanks a lot for your help, really appreciate it! We've added your advice to our backlog and will resolve it soon.

And for the issue mentioned in part 3, id for SQL document should be string with length, you may refer [here](https://learn.microsoft.com/en-us/rest/api/cosmos-db/create-a-document#body) for more info.

**Comment by dinbtechit** [comment]
@Flanker32 Absolutely. I also do Intellij Plugin development. I'll see if I can contribute in any way.

**Here are couple of my plugins.**
https://github.com/dinbtechit/vscode-theme
https://github.com/dinbtechit/intellij-elasticsearch-plugin

Thank you for pointing out the issue in part 3. That's good to know that the ID must be a string with a certain length. The error message was not obvious at all.
